src/src/components/PieChart.vue

```diff
@@ -69,28 +79,30 @@ export default {
         //.ease(d3.easeBounceIn)
 
 
-
+      config.fontsize = config.fontsize *scale;
```

Comment:
Do not modify config directly because it will change the value of config.fontsize permanently.